### ISSUE TITLE
LANG attribute required on text that is in a language not matching the page language

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#1350]
A LANG attribute is required on text that is in a language not matching the page language. For example the header "Francais" link should have a LANG="fr". Also the form-builder translation section should have text that is not the page language with a LANG attribute.

Requirement: [WCAG 3.2.1](https://w3c.github.io/wcag/21/guidelines/#language-of-parts)

**Comment by timarney** [comment]
@thiessenp-cds believe this one is done now correct?

**Comment by thiessenp-cds** [comment]
@timarney I think @srtalbot wanted to come back to this later when the translations were complete. The PR work is done though - yup :)
